File: leetcode/editor/en/[1150]Check If a Number Is Majority Element in a Sorted Array.java

//leetcode submit region begin(Prohibit modification and deletion)
class Solution {
    public boolean isMajorityElement(int[] nums, int target) {
        int count = 0;
        for (int num : nums) {
            if (num == target) {
                count++;
            }
        }
        return count > (nums.length / 2);
    }
}
//leetcode submit region end(Prohibit modification and deletion)

File: editor/en/[1150]Check If a Number Is Majority Element in a Sorted Array.js
//leetcode submit region begin(Prohibit modification and deletion)
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {boolean}
 */
var isMajorityElement = function (nums, target) {
    let count = 0;

    for (const num of nums) {
        if (num === target) {
            count++;
        }
    }

    return count > Math.floor(nums.length / 2);
};
//leetcode submit region end(Prohibit modification and deletion)
